src/app/+search-page/search-page.module.ts

```diff
@@ -65,15 +63,14 @@ const effects = [
     SearchBooleanFilterComponent,
   ],
   providers: [
-    SearchService,
     SearchSidebarService,
     SearchFilterService,
     SearchConfigurationService
   ],
   entryComponents: [
-    ItemSearchResultListElementComponent,
-    CollectionSearchResultListElementComponent,
-    CommunitySearchResultListElementComponent,
+    // ItemSearchResultListElementComponent,
+    // CollectionSearchResultListElementComponent,
+    // CommunitySearchResultListElementComponent,
```

Comment:
Same here. Unsure if these should just be removed or not.

Reply:
Removed commented out declarations